hi-ml-cpath/src/health_cpath/datasets/tcga_prad_dataset.py

```diff
@@ -11,6 +11,9 @@
 from health_cpath.datasets.base_dataset import DEFAULT_LABEL_COLUMN, SlidesDataset
 
 
+TCGA_PRAD_DATASET_FILE = "dataset.csv"
```

Comment:
is this necessary? we can use DEFAULT_CSV_FILENAME from base_dataset? let's try to limit duplicated constants

Reply:
Similar comment here: yes, the constant happens to have the same value, but it's use may be different. I'd use the same
constant for everything that MUST be changed together. For this constant, it is used in the script that writes the
dataset and here where we read it.